src/flow/connect/connectSettings.js

```diff
@@ -23,7 +25,15 @@ export default connect(
     var contacts = state.events.contacts
     const calendars = state.events.calendars
     const user = state.auth.user
-    return { show, contacts, calendars, addCalendarUrl, user }
+    const verifyAddCalendarData = state.events.verifyAddCalendarData
```

Comment:
variable name should not describe an action but an object

Reply:
like `verifiedNewCalendarData`

Reply:
Changed.

Reply:
that was quick!